### PR TITLE
Share Fabric app fastlane files 🚀

### PR DESCRIPTION
**Pull request by JaviSoto**

### CODE REVIEW

File: Fabric/Appfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for_lane :enterprise do
+    app_identifier "io.fabric.enterprise"
```
Comment (KrauseFx) [inline Fabric/Appfile]: This should be 2 spaces

Comment (KrauseFx) [inline Fabric/Appfile]: Same for all files actually

File: Fabric/Fastfile [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+    def tag_name(build_type, version_number, build_number)
+        "#{build_type}/#{version_number}/#{build_number}"
+    end
+
+    def load_keys
+        begin
+            import("keys")
+        rescue => exception
+            Helper.log.error("fastlane/keys file not found. To use this Fastfile, copy fastlane/keys.template into fastlane/keys and fill in the values".red)
```
Comment (KrauseFx) [inline Fabric/Fastfile]: You can use `UI.error` instead, then you don't need the `.red`

File: Fabric/Fastfile [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    private_lane :rocket do
+        puts "
+        /\\
+       |==|
+       |  |
+       |  |
+       |  |
+      /____\\
+      |    |
+      |fab |
```
Comment (KrauseFx) [inline Fabric/Fastfile]: :heart: This should be a built-in action, please submit a separate PR to fastlane adding this :rocket:

File: README.md [hunk 4]
```diff
@@ -250,6 +250,25 @@ A description on how the setup works is available in the [Overview](/Touchwonder
 </p>
 ----
 
+
+### [Fabric app](https://fabric.io/home)
```
Comment (KrauseFx) [inline README.md]: You can add this right below Fitbay instead :+1:

**Comment by KrauseFx** [comment]
Thanks for the PR, I added some comments. Especially please fix the spacing to be 2 spaces :rocket:

File: Fabric/Fastfile (hunk 3 above)
Comment (JaviSoto) [inline Fabric/Fastfile]: Haha! Will do lol

**Comment by JaviSoto** [comment]
Done :) let me know if everything looks good.

**Comment by KrauseFx** [comment]
This is great, thank you for cleaning things up! :+1:

**Comment by KrauseFx** [comment]
I hope you transferred the changes to your code base too? :wink:
